### PR TITLE
Add: zeal layer

### PR DESCRIPTION
**Pull request by chenlijun99**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

Dash works only on Mac

[Please explain **in detail** why the changes in this PR are needed.]

### CODE REVIEW

**Comment by chenlijun99** [comment]
BTW, is there a way to distinguish between visual mode and normal mode mapping in the guide buffer?
So when I'm in visual mode only "vmap" or "vnoremap" will be shown and the same for the other modes.

For example for the following mappins

```vimscript
  call SpaceVim#mapping#space#def('nmap', ['D', 'd'],
        \ '<Plug>Zeavim', 'search word under cursor', 0)
  call SpaceVim#mapping#space#def('vmap', ['D', 'D'],
        \ '<Plug>ZVVisSelection', 'search selected text', 0)
  call SpaceVim#mapping#space#def('nmap', ['D', 's'],
        \ '<Plug>ZVKeyDocset', 'specify docset and query', 0)
```

it would be handy to have `nmap Dd` and `vmap Dd`, instead of `nmap Dd` and `vmap DD`.

File: docs/layers/tools/zeal.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<!-- vim-markdown-toc -->
+
+## Description
+
+This layer provides Zeal integration for SpaceVim.
+
+## Layer Installation
+
+To use this configuration layer, add it to your `~/.SpaceVim.d/init.vim`.
```
Comment (wsdjeg) [inline docs/layers/tools/zeal.md]: It should be `init.toml`.

Comment (chenlijun99) [reply docs/layers/tools/zeal.md]: Fixed

**Comment by wsdjeg** [comment]
merged, thanks.

**Pull request merged**